views/index.html

```diff
@@ -1,21 +1,23 @@
 <!DOCTYPE html>
 <html lang="en">
-	<head>
-		<meta charset="utf-8">
-		<meta name="viewport" content="width=device-width,initial-scale=1">
-		<title>Hello HTML</title>
-		<link rel="stylesheet" href="/style.css">
-	</head>
-	<body>
-		<h1>Hello, HTML.</h1>
-		<p>Do not use until Challenge #12</p>
-     <form action="/name" method="post">
-      <label>First Name :</label>
-      <input type="text" name="first" value="John"><br>
-      <label>Last Name :</label>
-      <input type="text" name="last" value="Doe"><br><br>
-      <input type="submit" value="Submit">
-    </form> 
-	</body>
 
-</html>
\ No newline at end of file
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <title>Hello HTML</title>
+    <link rel="stylesheet" href="/style.css">
+</head>
+
+<body>
+    <h1>Hello, HTML.</h1>
+    <p>Do not use until Challenge #12</p>
+    <form action="/name" method="post">
+        <label>First Name :</label>
+        <input type="text" name="first" value="John"><br>
+        <label>Last Name :</label>
```

Comment:
I would remove the spaces from before the colons.  It's a bit more standard to have the space follow the colon rather than precede it. 
IE: "Label: " rather than "Label :"

Reply:
ok thank you for suggestion.